modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -403,6 +446,80 @@ Users who can access the Windows Network Drive storage via share won't receive a
+
+. As an admin, goto menu:Settings[Admin > Storage] and create a new CWND based mount point.
++
+.Figure 5. Add a Collaborative Windows Network Drive Mount
+image:enterprise/external_storage/windows_network_drive/cwnd_add_storage.png[Add Collaborative Windows Network Drive Mount, width=50%]
+. Chose any name for the mount point that fits your needs.
+. Select User Login Type.
++
+[IMPORTANT]
+====
+Only the following three are valid selections for CWND
+[loweralpha]
+. `Log-in credentials, saved in session`
+. `Log-in credentials, saved in database`
+. `User entered, stored in database` ^[1]^
++
+[1] Must be used in case user authentication is made with OIDC
+====
++
+.Figure 6. Select How User Logs Into the Mount Point
+image:enterprise/external_storage/windows_network_drive/cwnd_login_possibilities.png[Select How User Logs Into the Mount Point]
+[loweralpha]
+.. `Log-in credentials, saved in session`
++
+When the user logs into ownCloud via browser, the credentials to authenticate CWND are taken from this login. These credentials immediate end when the user logges out because the session has ended. This may impact client synchronisation or any access to other users via sharing. In case, no synchroinisation can be processed and sharees will lose access tho their shared resource when the sharer logges out. _This login type is by design not compatible with OIDC authentication._
+.. `Log-in credentials, saved in database`
++
+Similar to `Log-in credentials, saved in session`, the credentials to authenticate CWND are taken from the login but saved into the ownCloud database. Any relogin also updates the database entry. As the credentials to access CWND are taken from the database, a user logout will not stop CWND access and serving data is continued. _This login type is by design not compatible with OIDC authentication._
+.. `User entered, stored in database`
++
+User login to ownCloud and providing credentials to access the CWND mount are completely separated. The user may see post login to ownCloud his CWND mounts being inaccessible. To regain access, a pop up will appear when clicking on the mount point and the user must enter his share credentials which are then stored into the ownCloud database. As the credentials to access CWND are taken from the database, a user logout will not stop CWND access and serving data is continued. _This login type is by design *the only compatible with OIDC authentication*._
```

Comment:
We might want to skip the part about the popup. I don't think it will work properly with CWND.
I think the complain will come from a wrong service account password (usually), but the credentials we'll update are the user's ones, which might be right.

Reply:
The SA pwd has not changed and is independent of the users pwd. The SA still can access the resource. But as userA eg. has not logged in before in oC, there are no credentials for this mount for userA present - the mount will turn unavailable for userA. userA must now enter the credentials. Q: how will userA provide the credentials to regain access ?

Reply:
In the personal storage of the user, he should see a section to add the credentials for the mount. I think the admin needs to enable the personal storages though.

Reply:
THIS IS NEW - NEEDS CLARIFICATION
@jnweiger fyi

Reply:
admin user:
![Screenshot from 2021-05-07 10-47-27](https://user-images.githubusercontent.com/1477829/117424125-c0e5cf80-af21-11eb-9fd9-54f26b174cb3.png)

target user:
![Screenshot from 2021-05-07 10-47-57](https://user-images.githubusercontent.com/1477829/117424167-ca6f3780-af21-11eb-97b9-c4918f3bf7b0.png)

The user would need to fill the credentials for the mount point. He can use any valid SMB account.

Reply:
fixed